### ISSUE TITLE
Support animated gifs/webp

### ISSUE THREAD

**Issue by Stebalien** [alphapapa/ement.el#160]
(optionally, of course)

Putting point on an image and running `(image-animate (image--get-image))` will cause it to animate once, so this is clearly possible. Likely slightly annoying and difficult to make performant... but it's something to put on the TODO list.

**Comment by alphapapa** [comment]
Yeah, I wasn't sure if trying to make animations work in Emacs was a good idea.  I wouldn't mind if it were an option, assuming it doesn't break anything.

Not sure if WEBP is even supported in Emacs.  Maybe Emacs 29/30?

**Comment by Stebalien** [comment]
It's supported in emacs 29.

**Comment by alphapapa** [comment]
I guess it should just work when running on Emacs 29, then, right?

**Comment by Stebalien** [comment]
Webp, yes. But you have to tell emacs to animate images if you want that (and, IIRC, you have to keep telling it to loop them).

**Comment by alphapapa** [comment]
Yeah, that part I'll want to make optional.  Thanks.
